tests/cypress/support/commands_canvas3d.js

```diff
@@ -11,16 +11,13 @@ Cypress.Commands.add('compareImagesAndCheckResult', (baseImage, afterImage, noCh
 });
 
 Cypress.Commands.add('create3DCuboid', (cuboidCreationParams) => {
-    cy.get('.cvat-draw-cuboid-control').trigger('mouseover');
-    cy.get('.cvat-draw-cuboid-popover-visible').find('[type="search"]').click({ force: true });
-    cy.get('.ant-select-dropdown')
-        .not('.ant-select-dropdown-hidden')
-        .within(() => {
-            cy.contains(new RegExp(`^${cuboidCreationParams.labelName}$`)).click();
-        });
+    cy.interactControlButton('draw-cuboid');
+    cy.switchLabel(cuboidCreationParams.labelName, 'draw-cuboid');
     cy.get('.cvat-draw-cuboid-popover-visible').find('button').click();
     cy.get('.cvat-canvas3d-perspective')
         .trigger('mousemove', cuboidCreationParams.x, cuboidCreationParams.y)
         .dblclick(cuboidCreationParams.x, cuboidCreationParams.y);
     cy.wait(1000); // Waiting for a cuboid creation
+    cy.get('.cvat-draw-cuboid-popover-visible').should('not.exist');
+    cy.get('.cvat-draw-cuboid-popover').should('be.hidden');
```

Comment:
I am not sure we should test that the popovers are closed now.

Reply:
Without this check, there is a high probability of getting error ```AssertionError: Timed out retrying after 25000ms: Expected to find element: `.*-popover-visible`, but never found it.```